ui/raidboss/data/06-ew/raid/p10n.ts

```diff
@@ -1,14 +1,201 @@
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  silkspitOnYou: boolean;
+}
+
+const bossNameUnicode = 'Pand\u00e6monium';
 
 const triggerSet: TriggerSet<Data> = {
   id: 'AnabaseiosTheTenthCircle',
   zoneId: ZoneId.AnabaseiosTheTenthCircle,
   timelineFile: 'p10n.txt',
-  triggers: [],
+  initData: () => {
+    return {
+      silkspitOnYou: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'P10N Silkspit You',
+      type: 'HeadMarker',
+      netRegex: { id: '01CE' },
+      condition: (data, matches) => data.me === matches.target,
+      alertText: (data, _matches, output) => {
+        data.silkspitOnYou = true;
+        return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Away from party and posts',
+        },
+      },
+    },
+    {
+      id: 'P10N Silkspit Others',
+      type: 'HeadMarker',
+      netRegex: { id: '01CE', capture: false },
+      delaySeconds: 0.2,
+      suppressSeconds: 1,
+      infoText: (data, _matches, output) => {
+        if (data.silkspitOnYou === false)
+          return output.text!();
+      },
+      outputStrings: {
+        text: {
+          en: 'Away from marked players',
+        },
+      },
+    },
+    {
+      id: 'P10N Pandaemonic Pillars',
+      type: 'StartsUsing',
+      netRegex: { id: '825D', source: bossNameUnicode, capture: false },
+      durationSeconds: 5,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Soak towers',
+        },
+      },
+    },
+    {
+      id: 'P10N Imprisonment',
+      type: 'StartsUsing',
+      netRegex: { id: '8262', source: bossNameUnicode, capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Avoid jails',
+        },
+      },
+    },
+    {
+      id: 'P10N Cannonspawn',
+      type: 'StartsUsing',
+      netRegex: { id: '8264', source: bossNameUnicode, capture: false },
+      suppressSeconds: 1,
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Under jails',
+        },
+      },
+    },
+    {
+      id: 'P10N Ultima',
+      type: 'StartsUsing',
+      netRegex: { id: '827B', source: bossNameUnicode, capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'P10N Pandaemoniac Meltdown',
+      type: 'StartsUsing',
+      netRegex: { id: '8276', source: bossNameUnicode, capture: false },
+      response: Responses.stackMarker(),
```

Comment:
A tiny and ignorable nit: In my logs, at least, it looks like there's an associated `87A2` Ability line on the actual target of this spell. It goes out at the same time as `8276` starts casting. Could we use it instead and switch the response to `stackMarkerOn()`?

Reply:
Done.  Thanks for the suggestion!